docs/sources/reference/api.md

```diff
@@ -1,69 +1,81 @@
 ---
-title: Grafana Loki HTTP API
-menuTitle: "HTTP API"
-description: Loki exposes REST endpoints for operating on a Loki cluster. This section details the REST endpoints.
+title: Loki HTTP API
+menuTitle: HTTP API
+description: Loki exposes HTTP endpoints for data ingestion, data retrieval, as well for cluster management.
 aliases:
 - ../api/
 weight: 100
 ---
 
-# Grafana Loki HTTP API
+# Loki HTTP API
 
-Grafana Loki exposes an HTTP API for pushing, querying, and tailing log data.
-Note that authenticating against the API is
-out of scope for Loki.
+Loki exposes an HTTP API for pushing, querying, and tailing log data, as well
+as for viewing and managing cluster information.
 
-## Microservices mode
+**Note that authorization is not part of the Loki API.**
+Authorization needs to be done separately, e.g. using an open-source load-balancer such as NGINX.
 
-When deploying Loki in microservices mode, the set of endpoints exposed by each
-component is different.
+## Endpoints
```

Comment:
I wonder whether we should structure the endpoints overview like in the [Mimir docs](https://grafana.com/docs/mimir/latest/references/http-api/#endpoints).

Reply:
At first glance I can't tell how it's organized. It doesn't seem to be alphabetized on ANY of the three columns.  Looking closer, it seems to be loosely alphabetized by Endpoint (the third column), which isn't what I would have expected.

I really liked how you chose to structure the endpoints, it seemed very logical to me.  Let's get some feedback from users before we change anything.